### PR TITLE
CommandManager.xml: event is weak because of custom implementation, not because of being static

### LINKED ISSUE #4235: Mistake that static events are weak
# Mistake that static events are weak

There is a mistake on this page: https://docs.microsoft.com/en-us/dotnet/api/system.windows.input.commandmanager.requerysuggested?view=netframework-4.7.1
It says "Since this event is static, it will only hold onto the handler as a weak reference" that is mistake.
Static events has only one difference from instance events - they are static.

### PR DESCRIPTION
**Pull request by pkulikov**
Updated phrasing of RequerySuggested event: it is weak not because of being static.

Fixes #4235 

## Suggested Reviewers
@svick @jinek what do you think of updated phrasing

I would not mention `WeakEventManager` since there were too much implementation details in the documentation.

### CODE REVIEW

**Review by jinek — APPROVED** [review]
Thank you!

File: xml/System.Windows.Input/CommandManager.xml [hunk 1]
```diff
@@ -567,7 +567,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Since this event is static, it will only hold onto the handler as a weak reference.  Objects that listen for this event should keep a strong reference to their event handler to avoid it being garbage collected. This can be accomplished by having a private field and assigning the handler as the value before or after attaching to this event.  
+ This event is implemented in such a way that it only holds onto the handler as a weak reference.  Objects that listen for this event should keep a strong reference to their event handler to avoid it being garbage collected. This can be accomplished by having a private field and assigning the handler as the value before or after attaching to this event.
```
Comment (rpetrusha) [inline xml/System.Windows.Input/CommandManager.xml]: Mentioning that it's implemented in such a way raises the question of how it's implemented, which is best avoided. It's much more straightforward here to simply say, "This event only hold on to the handler as a weak reference."

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for making this correction, @pkulikov. I've suggested alternate wording for the revised sentence to completely remove mention of implementation details.

File: xml/System.Windows.Input/CommandManager.xml (hunk 1 above)
Comment (pkulikov) [reply xml/System.Windows.Input/CommandManager.xml]: @rpetrusha thanks for the feedback, I've updated the sentence. Indeed, my first reaction on the issue was to check the reference source :).

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you, @pkulikov, for making the additional change. We'll merge your PR now. Your changes should appear live on docs.microsoft.com in the next day or two.

**Pull request merged**